ks_includes/locales/hu_HU/LC_MESSAGES/KlipperScreen.po

```diff
@@ -553,7 +555,9 @@ msgid "Z Calibrate"
 msgstr "Z Kalibrálás"
 
 msgid "Z Move Speed (mm/s)"
-msgstr "Z Mozgási sebesség (mm/s)"
+msgstr ""
+"Z Mozgási sebesség\n"
```

Comment:
do not put a \n
we have a line wrapper, if it's needed it's automatic

Reply:
Sorry! I did not notice. It could have happened here by accident. I'll fix it. Thanks! By: AntoszHUN